boards/ct/ctcc/ctcc_nrf52840.dts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2024 CTHINGS.CO
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+
+/ {
+	model = "CTHINGS.CO Connectivity Card nRF52840";
+	compatible = "ct,ctcc-nrf52840";
+
+	chosen {
+		zephyr,console = &cdc_acm_uart;
+		zephyr,shell-uart = &cdc_acm_uart;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+	};
+
+	leds {
+		 compatible = "gpio-leds";
+		 led1: led_1 {
+			 gpios = <&gpio0 23 GPIO_ACTIVE_LOW>;
+			 label = "LED 1";
+		 };
+		 led2: led_2 {
+			 gpios = <&gpio0 22 GPIO_ACTIVE_LOW>;
+			 label = "LED 2";
+		 };
+	};
+
+	aliases {
+		led0 = &led1;
+		led1 = &led2;
+		mcuboot-led0 = &led1;
+	};
+};
+
+&flash0 {
+
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		boot_partition: partition@0 {
+			label = "mcuboot";
+			reg = <0x000000000 0x00010000>;
+		};
+		slot0_partition: partition@10000 {
+			label = "image-0";
+			reg = <0x00010000 0x00068000>;
+		};
+		slot1_partition: partition@78000 {
+			label = "image-1";
+			reg = <0x00078000 0x00068000>;
+		};
```

Comment:
(non blocking) these sector sizes are not optimal, if you build MCUboot with debug logging it will print a message

Reply:
Right, I can see with max dbg logs it won't fit into flash. However I'd leave it up to user to change mcuboot partition if needed.

Reply:
well the sizes here should be good defaults, if using the default MCUboot configuration then slot0 should be 2 sectors larger than slot1

Reply:
I got it - I'll change it to the partition layout from `nrf52840dongle` `fstab-debugger.dtsi`